src/http/routes/object/copyObject.ts

```diff
@@ -37,15 +37,16 @@ export default async function routes(fastify: FastifyInstance) {
     '/copy',
     {
       schema,
+      config: {
+        operation: { type: 'object.copy' },
+        resources: (req: FastifyRequest<copyRequestInterface>) => {
+          const { sourceKey, destinationKey, bucketId, destinationBucket } = req.body
+          return [`${bucketId}/${sourceKey}`, `${destinationBucket || bucketId}/${destinationKey}`]
```

Comment:
Yes is possible to filter by arrays, @Ziinc told me he would implement regex matching too

Reply:
In this case wouldn't it be more explicit and clear if it was two string fields though? One for source and one for destination.
If it was a variable length array then chucking it all in an array would make sense for searching, but it isn't so for this case

Reply:
Hey @Ziinc the `resources` fields might be different for different operation, in this case the resources for this operations are the source and destination.

The reason i'd like to use a single field for this is because if I want to query any operation for a specific resource I can simply query the array and I will get all the logs for this specific resource.

Ex, it has been copied, uploaded, delete, moved, etc...